scripts/update

```diff
@@ -11,7 +11,7 @@ function cleanup() {
 
 trap cleanup ERR
 
-# Build app container image first
+# Build app container image
 docker compose build app
```

Comment:
I don't know why the app and db/django builds were separate, but I wonder if this can now just be `docker compose build`?

Reply:
I don't recall the reason exactly, but I do vague remember something about the frontend docker container needs to build first (frontend used to have a dockerfile for itself before this PR) and then build the static app, and then the backend (django, postgres) containers build, so that later the static asset collection could work etc.

We shouldn't have that issue anymore. Addressed in https://github.com/azavea/echo-locator/pull/666/commits/71f43fe95b69a09be52b5c9b5d650643ea7745e5